site/docs/hosting/firebase.md

```diff
@@ -0,0 +1,186 @@
+# Hosting: Firebase Functions
+
+This tutorial will guide you through the process of deploying your bot to [Firebase](https://firebase.google.com) using [Firebase Serverless Functions](https://firebase.google.com/docs/functions).
+
+## Prerequisites
+
+To follow this tutorial, you will need a Google account.
+If you don't have one, you can [create one here](https://accounts.google.com/signup).
+
+## Setup
+
+This section will guide you through the setup process.
+If you need a more detailed explanation of each step, you can refer to the [official Firebase documentation](https://firebase.google.com/docs/functions/get-started).
+
+### Create a Firebase Project
+
+1. Go to the [Firebase Console](https://console.firebase.google.com/) and click **Add Project**.
```

Comment:
```suggestion
1. Go to [Firebase Console](https://console.firebase.google.com/) and click **Add Project**.
```

Reply:
`Firebase Console` is a thing, semantically speaking (even though it is a website), so i would  prefer to keep the `the`

Reply:
```suggestion
1. Go to the [Firebase console](https://console.firebase.google.com/) and click **Add Project**.
```